tests/foreman/api/test_errata.py

```diff
@@ -204,24 +204,21 @@ def test_positive_install_in_hc(module_org, activation_key, custom_repo, default
         host_ids = [client.nailgun_host.id for client in clients]
         host_collection.host_ids = host_ids
         host_collection = host_collection.update(['host_ids'])
-        host_collection_query = f'host_collection_id = {host_collection.id}'
-        JobInvocation.create(
-            {
+        task_id = default_sat.api.JobInvocation().run(
+            data={
                 'feature': 'katello_errata_install',
-                'inputs': f'errata=\'{CUSTOM_REPO_ERRATA_ID}\'',
-                'search-query': host_collection_query,
-                'organization-id': f'{module_org.id}',
-            }
+                'inputs': {'errata': f'{CUSTOM_REPO_ERRATA_ID}'},
+                'targeting_type': 'static_query',
+                'search_query': f'host_collection_id = {host_collection.id}',
+                'organization_id': module_org.id,
+            },
+        )['id']
+        wait_for_tasks(
+            search_query=('label = Actions::RemoteExecution::RunHostsJob' f' and id = {task_id}'),
```

Comment:
Formatting of this string is screwy, `Black` likely formatted after you wrote the lines. 

Put the `f` in front of the entire string, and make it a single string.  (Right now its two strings)